docker/Dockerfile.mediamtx

```diff
@@ -1,17 +1,23 @@
-FROM ubuntu:24.04
+FROM	ubuntu:24.04
 
-ENV MEDIAMTX_VERSION="v1.9.3"
+ENV	MEDIAMTX_VERSION="1.9.3"
+
+ADD	"https://github.com/bluenviron/mediamtx/releases/download/v${MEDIAMTX_VERSION}/mediamtx_v${MEDIAMTX_VERSION}_linux_amd64.tar.gz"	/opt/mediamtx.tar.gz
```

Comment:
Nice!
I would move it down below `apt-get install` as it may change more often than system packages.
And being lower on the dockerfile would make cached layers being used more optimally